src/main/java/com/openlattice/chronicle/ChronicleStudyApi.java

```diff
@@ -165,13 +221,15 @@ Void deleteStudyAndAllNeighbors(
     /**
      * Update chronicle_used_by associations when apps usage survey is submitted
      *
+     * @param organizationId     - Id of the organization to which study belongs
      * @param studyId            - the study id
      * @param participantId      - participantId
      * @param associationDetails - mapping from association EKID to association entity data
      * @return number of updated associations
      */
-    @POST( BASE + PARTICIPANT_PATH + DATA_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS )
+    @POST( BASE + ORGANIZATION_ID_PATH + STUDY_ID_PATH + PARTICIPANT_ID_PATH + APPS )
```

Comment:
This looks like a breaking change.

Reply:
Correct. However, I removed the PARTICIPANT_PATH + DATA_PATH because they don't make sense being part of the the path, unlike in the endpoints that are for downloading participant's data (raw, preprocessed and usage). Also updated the web app since to use the corrected path.